### ISSUE TITLE
Event uuid collision with deleted events?

### ISSUE THREAD

**Issue by parosio** [MISP/PyMISP#1308]
I'm testing a python script that queries a remote MISP instance (for a limited set of events) and creates the events on a local MISP instance.
If I delete an event from the local instance (`https://local_url/events/delete/event_id`), further attempts to recreate it (using `PyMISP(local_url, local_key, secure).add_event(evt2load, metadata=True)` ) fail with:
`{'errors': (403, {'saved': False, 'name': 'Could not add Event', 'message': 'Could not add Event', 'url': '/events/add', 'errors': []})}`
I can't find any error in local error.log file.

In order to recreate the event I must unset the event uuid before calling add_event:
<code>
orig_uuid = evt2load.pop('uuid')
PyMISP(local_url, local_key, secure).add_event(evt2load, metadata=True)
</code>

It seems there's a uuid collision of some sort even if the local event has been deleted. 
Is this an expected behavior?

python3.10.11
pymisp 2.5.1
MISP 2.4.124

**Comment by Rafiot** [comment]
Yes, it is. When an event is deleted from MISP, its UUID is added in the event blacklist (this is to avoid re-syncing events from other instances).

**Comment by Lobasin** [comment]
Hello can anyone help resolved my submission ready to collaborate from BugCrowd, who’s in..
